### PR TITLE
style: updated Hero sections to fix inconsistencies

### PR DESCRIPTION
**Pull request by atepem**
## Proposed changes

Some changes to Hero sections for pages bonds, build, contribute, govern, launch and operate responsiveness - to be more consistent throughout.

## Screenshots/Recordings
Main changes on these pages; before:
build page before
contribute page before
govern page before

After:
build page after
contribute page after
govern page after

### CODE REVIEW

**Review by oaksprout — DISMISSED** [review]

File: package-lock.json
[the remark this answers is not in the document]
Comment (atepem) [reply package-lock.json:1]: Thanks for letting me know! I've deleted and pushed it now :)

**Comment by Tanya-atatakai** [comment]
@atepem can you please look at the build page, as Mohan mentioned, the image is too big, but also I think that another problem might be in buttons not wrapped in small screens https://tailwindcss.com/docs/flex-wrap

image

File: components/OperatePage/Hero.jsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const Hero = () => (
         </div>
       </div>
 
-      <div className="hidden sm:block col-span-1" />
+      {/* <div className="hidden sm:block col-span-1" /> */}
```
[the remark this answers is not in the document]
Comment (atepem) [reply components/OperatePage/Hero.jsx]: This has been removed :)

File: yarn.lock
[the remark this answers is not in the document]
Comment (atepem) [reply yarn.lock:1]: I've run yarn install again, hopefully that should be good now...

**Comment by atepem** [comment]
> @atepem can you please look at the build page, as Mohan mentioned, the image is too big, but also I think that another problem might be in buttons not wrapped in small screens https://tailwindcss.com/docs/flex-wrap
> 
> image

I've fixed the resources sizing issue in the /contribute page and /build page should work better now with smaller screens, although the buttons will be stacked on top of each other instead of side by side. 
This is with a 375x667 screen (Iphone SE):
image
As opposed to a full monitor:
image

**Review by Tanya-atatakai — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Pull request merged**